activerecord/lib/active_record/railties/databases.rake

```diff
@@ -158,6 +158,20 @@ db_namespace = namespace :db do
     puts "Current version: #{ActiveRecord::Migrator.current_version}"
   end
 
+  namespace :version do
+    desc 'Prints the current schema version number'
+    task :number => [:environment, :load_config] do
+      puts ActiveRecord::Migrator.current_version
+    end
+
+    desc "Prints the file path of current schema's migration"
+    task :file => [:environment, :load_config] do
+      current_version = ActiveRecord::Migrator.current_version
+      migration_file = Dir.glob("#{ActiveRecord::Migrator.migrations_path}/*#{current_version}*.rb").first
```

Comment:
Instead of `/*#{current_version}*.rb`, can't we use `/#{current_version}_*.rb`? It seems improbable to have a migration-number-looking-thing in the middle of a migration's filename, but not impossible.